app/client/components/analytics.tsx

```diff
@@ -78,11 +79,23 @@ export class AnalyticsTracker extends React.PureComponent<{}> {
   constructor(props: {}) {
     super(props);
     if (typeof window !== "undefined" && window.ga) {
-      this.INTCMP = parse(window.location.href, true).query.INTCMP;
+      const queryParams = parse(window.location.href, true).query;
+
+      this.INTCMP = queryParams.INTCMP;
 
       if (window.guardian) {
         // tslint:disable-next-line:no-object-mutation
         window.guardian.INTCMP = this.INTCMP;
+
+        const abName = queryParams.abName;
+        const abVariant = queryParams.abVariant;
+
+        if (abName && abVariant) {
+          window.guardian.abTest = {
```

Comment:
I'm missing a bit of context here... but is it safe to assume that a user will only ever be part of one AB test when they hit /manage?

Reply:
I think so. My understanding from talking with @twrichards: AB tests in manage are done through Optimize. So this AB test relates to ab tests that were initiated outside of the manage platform. 

Examples:
- payment failure emails
- payment failure banners

Maybe some context as a comment would be useful?

Reply:
I think this is fine as is, I just wanted to make sure I'd understood correctly 👍